sql-cli/sql_cli/project.py

```diff
@@ -99,30 +94,6 @@ def _initialise_global_config(self) -> None:
             "airflow", "dags_folder", str(self._airflow_dags_folder.resolve()), global_env_filepath
         )
 
-    def transform_env_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
```

Comment:
At the moment this method was only expanding host paths for SQLite connection and persisting back the expanded host path to the YAML. Persisting back is risky. Instead we should resolve this every time in the config instance alone without persisting back to the YAML and the logic to transform the connection to resovle host path has been moved to the `connections.py` module's method `convert_to_connection`